isb_lib/sesar_adapter.py

```diff
@@ -156,25 +158,32 @@ def _validateResolvedContent(thing: Thing):
 
 def reparseRelations(thing: Thing, as_solr: bool = False):
     _validateResolvedContent(thing)
-    item = SESARItem(thing.id, thing.resolved_content)
-    if as_solr:
-        return item.solrRelations()
-    return item.asRelations()
+    if thing.id is not None and thing.resolved_content is not None:
+        item = SESARItem(thing.id, thing.resolved_content)
+        if as_solr:
+            return item.solrRelations()
+        return item.asRelations()  # type: ignore
```

Comment:
This method appears to have disappeared.  This code isn't used anymore, I don't think.  It was the original relations solr schema that we aren't using anymore.